Docs: update geomap docs to include Base layer in the four Data layer types list (#50874)
grafana/grafana: docs/sources/visualizations/geomap.md

## Base layer

> The base layer loads in a blank world map from the tile server to the Grafana panel. Several base layer options are available each with specific configuration options to style the base map. Base layers can be added to the Data layer stack using layer controls in the Data layer panel option, or configured separately using the Base layer panel option. Adding multiple base layers to the Data layer stack allows you to overlay them, you can use the opacity controls to balance the overlay mix. The default base layer is CartoDB base map. Custom default base layers can be defined in the `.ini` configuration file.

Reviewer: ```suggestion
The base layer loads in a blank world map from the tile server to the Grafana panel. Several base layer options are available each with specific configuration options to style the base map. Base layers can be added to the Data layer stack using layer controls in the Data layer panel option, or configured separately using the Base layer panel option. To overlay layers, add multiple base layers to the Data layer stack, and use the opacity controls to balance the overlay mix. The default base layer is CartoDB base map. Custom default base layers can be defined in the `.ini` configuration file.
```

}`
```

> `enable_custom_baselayers` allows you too enable or disable custom open source base maps that are already implemented. The default is `true`.

Reviewer 2: This looks incorrect. Should be `to` and not `too` as it allows you **to** enable or disable.

Author: 😅 I confuse myself sometimes! Thanks!
